specification/sql/resource-manager/readme.md

```diff
@@ -276,6 +276,7 @@ input-file:
 - ./Microsoft.Sql/preview/2021-11-01-preview/ManagedDatabaseSensitivityLabels.json
 - ./Microsoft.Sql/preview/2021-11-01-preview/ManagedServerDnsAliases.json
 - ./Microsoft.Sql/preview/2021-11-01-preview/SensitivityLabels.json
+- ./Microsoft.Sql/preview/2021-11-01-preview/IPv6FirewallRules.json
```

Comment:
we need to replace the 2011-08-01-preview one in line 274 to pass the check in pipelines